dipy/tracking/local_tracking.py

```diff
@@ -31,8 +31,9 @@ def _get_voxel_size(affine):
         return np.sqrt(dotlin.diagonal())
 
     def __init__(self, direction_getter, stopping_criterion, seeds, affine,
-                 step_size, max_cross=None, maxlen=500, fixedstep=True,
-                 return_all=True, random_seed=None, save_seeds=False):
+                 step_size, max_cross=None, maxlen=500, minlen=0,
```

Comment:
As you can see in this link: https://github.com/dipy/dipy/blob/eede25d73dcff7fff758d64dbdb829533de6a05a/dipy/tracking/local_tracking.py#L172

`minlen` was 1.  I would recommend to keep this value to avoid breaking previous code

Reply:
Good point, actually I am going to set it to 2 because I am now using `>=` instead of `>`.